External/FEXCore/Source/Interface/HLE/Thunks/Thunks.cpp

```diff
@@ -67,6 +69,107 @@ extern char __start_HostToGuestTrampolineTemplate[];
 extern char __stop_HostToGuestTrampolineTemplate[];
 
 namespace FEXCore {
+  namespace VDSO {
```

Comment:
This should be moved to some VDSO file, and should live in the linux frontend, as it is specific to that